src/LibraryManagement.Application/CommandHandlers/Authentication/LoginCommandHandler.cs

```diff
@@ -0,0 +1,75 @@
+﻿using LibraryManagement.Application.Commands.Authentication;
+using LibraryManagement.Domain.Entities.Authentication;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace LibraryManagement.Application.CommandHandlers.Authentication;
+
+public class LoginCommandHandler : IRequestHandler<LoginCommand, string>
+{
+    private readonly UserManager<User> _userManager;
+    private readonly IConfiguration _configuration;
+    private User? _user;
```

Comment:
I don't know that you use "?" for what

Reply:
I use the User? because the FindByEmailAsync (line 27) return the User? type.

Reply:
Ok I got it